apps/studio/prisma/seed.ts

```diff
@@ -216,6 +217,7 @@ async function main() {
       return db
         .insertInto('User')
         .values({
+          id: cuid2.createId(),
```

Comment:
`cuid` is generated by prisma using js code but `kysely` sees the raw db type - which doesn't guarantee this. 

hence, i have to use `cuid2` to generate it myself on seed (essentially what prisma is doing AHHAHA)